tunnistamo/settings.py

```diff
@@ -175,14 +223,14 @@
     'sass_processor.finders.CssFinder',
 )
 
-STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+STATIC_ROOT = env("STATIC_ROOT")
 STATIC_URL = '/sso/static/'
```

Comment:
That would indeed be nice, however, it seems like when using environ with this I'm getting:

```
tunnistamo-backend |     "You're using the staticfiles app "
tunnistamo-backend | django.core.exceptions.ImproperlyConfigured: You're using the staticfiles app without having set the required STATIC_URL setting.
```

So for now, this will be hardcoded and be figured out later hopefully 😄

Reply:
`STATIC_URL = env("STATIC_URL")` seems to work for me, however I don't think this should be configurable after all, because it seems to be hardcoded in `uwsgi.ini` (the same goes for `MEDIA_URL`). Also sass-processor seems to cause some strangeness when you change that setting.

So, if we agree that those should be hardcoded, would be good if you removed `STATIC_URL` from the env initialization at the top of the settings file, and from `docker-compose.env.yaml.template`.

Reply:
Fixed :)